OfficeUIFabric/Controls/MSButton.swift

```diff
@@ -57,17 +57,23 @@ open class MSButton: UIButton {
         }
     }
 
-    @objc public init(style: MSButtonStyle = .secondaryOutline) {
+    @objc public convenience init(style: MSButtonStyle = .secondaryOutline) {
+        self.init(frame: .zero)
         self.style = style
-        super.init(frame: .zero)
-        initialize()
+
+        update()
     }
 
     public required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
         initialize()
     }
 
+    public override init(frame: CGRect) {
```

Comment:
Please move it up above `init(coder:)` since this is more "important" initializer.

Reply:
I just moved it. And also the `init(style:)` implementation was reversed to its original form.